sdk/v2/core/workers.go

```diff
@@ -98,6 +98,17 @@ func WorkerPhasesTerminal() []WorkerPhase {
 	}
 }
 
+// IsTerminal returns a bool indicated whether or not the provided WorkerPhase
+// is a terminal phase
+func IsTerminal(phase WorkerPhase) bool {
+	for _, terminalPhase := range WorkerPhasesTerminal() {
+		if phase == terminalPhase {
+			return true
+		}
+	}
+	return false
+}
+
```

Comment:
Something like this is already in the `v2` branch. You can use `phase.IsTerminal()`

Reply:
Oh great!  Not sure how I missed this :)

Reply:
It was a very recent addition.

Reply:
The original `IsTerminal()` method exists in the internal pkg, so I copied the same over into the external sdk pkg -- or should I also remove the original instance in internal and update references?

Reply:
Ooooooh. I forgot that this component would obviously be using the SDK and not types internal to the apiserver.

The copy is good. I know it doesn't seem DRY, but I like the share-nothing approach between SDK and apiserver. Since we anticipate SDKs being implemented in many different languages (a TS/JS one is already under construction), the share-nothing approach seems even more appropriate because then the Go SDK isn't special somehow.

👍